assets/base/i18n/en.json

```diff
@@ -1177,6 +1177,7 @@
   "emoji_list.name": "Name",
   "emoji_list.search": "Search Custom Emoji",
   "emoji_list.somebody": "Somebody on another team",
+  "error.network_request_failed": "Cannot connect to the server. Please check your server URL and internet connection.",
```

Comment:
@jarredwitt That sounds good to me. How about `mobile.routes.select_server.server_ping_error`? Because we won't necessarily have gotten a response.